### PR TITLE
Additional updates to README.md

### PR DESCRIPTION
**Pull request by beav**
This patch documents fixes for two minor issues that I hit:

 * low disk space on /home partition
 * assumption that you are SSHing as vagrant user

I also added an example optional plugin config for helpful customizations that
developers may want to enable.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -190,6 +230,18 @@ sudo yum install nfs-utils
 sudo service start nfs-server
 ```
 
+#### low disk space
```
Comment (stbenjam) [inline README.md]: Nitpick, upper case L

Comment (stbenjam) [inline README.md]: Eh, nevermind, all the other titles are lowercase in the troubleshoot section.

**Comment by stbenjam** [comment]
Thanks @beav!

**Comment by ehelms** [comment]
Some of these seem like reasonable things to potentially include by default or as an easy to configure option by developers. Is there any reason not to do that for these? I always tend to prefer making things real versus documenting how to (since then its in code and works).

**Comment by stbenjam** [comment]
Oh that sounds like a good idea to me, the host plugin especially I think is pretty useful

**Comment by beav** [comment]
@ehelms that is a good idea, but I would like to get the gem cache to work first. I hit an issue that might be the same as https://github.com/fgrehm/vagrant-cachier/issues/74 (haven't confirmed or ruled it out yet).

**Pull request merged**